src/main/resources/todos.mustache

```diff
@@ -36,4 +36,29 @@
             &copy 2019 @angieg0nzalez | 8th Light Apprenticeship
         </footer>
     </body>
+    <script>
+        document
```

Comment:
I wouldn't hold up this PR since the focus is on the Java code.  However I think it's possible this could try to look up the `#todo-form` element and add an event listener before the DOM is finished loading.  You might try wrapping all this code in 

```javascript
document.addEventListener("DOMContentLoaded", function(event) { 
  //do work
});
```

https://stackoverflow.com/questions/799981/document-ready-equivalent-without-jquery